install/installer.js

```diff
@@ -95,13 +92,6 @@ function updateInstructions() {
 function updateInterfaceOptions() {
     document.querySelectorAll('.interface').forEach(option => {
         option.classList.remove('disabled');
```

Comment:
Not sure we even need to do anything in this function any more -- things can no longer become disabled, so removing that tag will never do anything